spec/ics-2-consensus-verification/README.md

```diff
@@ -0,0 +1,310 @@
+---
+
+ics: 2
+title: Consensus Verification
+stage: draft
+category: ibc-core
+requires: 23
+required-by: 3
+author: Juwoon Yun <joon@tendermint.com>, Christopher Goes <cwgoes@tendermint.com>
+created: 2019-02-25
+modified: 2019-04-02
+
+---
+
+## Synopsis
+
+This standard specifies the properties that consensus algorithms of chains implementing IBC are
+expected to satisfy. The properties are needed for efficient and safe verification in the higher
+level protocol abstractions. The algorithm which uses these properties to verify substates of
+another chain is referred to as a "light client".
+
+## Specification
+
+### Motivation
+
+`FullNode`s are procedures running a `Consensus`. Given a `([Transaction], Commit)`, a
+`FullNode` can compute the result `ConsensusState` that the `Consensus` is expected to commit on
+with the same `[Transaction]`, if exists.
+
+`Blockchain` defines required properties of the blockchain on the network. The implementors can
+check whether the consensus that they are using is qualified to be connected to the network or
+not. If not, they can modify the algorithm or wrap it with additional logic to make it
+compatible with the specification. It also provides base layer for the protocol that the other
+components can rely on.
+
+### Desired Properties
+
+This standard specification provides secure layer to verify other chains' canonical headers,
```

Comment:
I think we need to more clearly articulate that there is an *underlying* consensus process going on, which light clients are attempting to verify the output of in a way where it is very difficult to convince a light client that a different consensus transcript occurred (when it did not actually occur)